Twitter/DetailViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  DetailViewController.swift
+//  Twitter
+//
+//  Created by victoria_ku on 6/11/17.
+//  Copyright © 2017 victoria_ku. All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+
+
+  @IBOutlet weak var tableView: UITableView!
+  var tweet: Tweet?
+
+
+    override func viewDidLoad() {
+      super.viewDidLoad()
+      tableView.delegate = self
+      tableView.dataSource = self
+      //tableView.register("ReplyCell", forCellReuseIdentifier: "ReplyCell")
+      navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Reply", style: .plain, target: self, action: #selector(ReplyButtonTapped))
+
+
+        // Do any additional setup after loading the view.
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+  func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    return 1
+  }
+  func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+    let cell = tableView.dequeueReusableCell(withIdentifier: "DetailCell", for: indexPath) as! DetailCell
```

Comment:
usually you don't want to force unwrap this.

we should always try optionally unwrap it